### PR TITLE
Miscellaneous (mostly style) fixes

### PR DESCRIPTION
**Pull request by dcermak**
I've started digging into the codebase  of rpm-py-installer and found some issues and one typo. This PR should simplifies some variable assignments and removes some dead/pointless branches.

### CODE REVIEW

**Comment by junaruga** [comment]
@dcermak thanks for your PR!

It seems the lint tasks are failed on CI. You can run below command on your local to fix it.

```
$ tox -e lint-py3 lint-py2
```

File: install.py [hunk 1]
```diff
@@ -164,11 +156,8 @@ def download_and_install(self):
             try:
                 self.installer.install_from_rpm_py_package()
                 return
-            except RpmPyPackageNotFoundError as e:
-                Log.warn('RPM Py Package not found. reason: {0}'.format(e))
-
-                # Pass to try to install from the source.
-                pass
+            except RpmPyPackageNotFoundError as exc:
```
Comment (junaruga) [inline install.py]: `exc` is better than `e`? Why? Is it more general in Python such as `sys.exc_info`?
https://docs.python.org/3.7/library/exceptions.html#Exception

Comment (junaruga) [reply install.py]: Your commit comment says "Change exception variable name to exc to adhere to Python's naming conventions".
How did you check it? By a tool? Is there the page where I can refer it?

Comment (junaruga) [reply install.py]: There are still remaining `as e` lines.

```
$ grep -rn 'except ' install.py 
32:        except InstallSkipError:
159:            except RpmPyPackageNotFoundError as exc:
227:    except ImportError:
414:            except RemoteFileNotFoundError:
428:            except RemoteFileNotFoundError as exc:
890:        except InstallError as exc:
979:            except RemoteFileNotFoundError:
1066:            except RemoteFileNotFoundError as e:
1136:        except InstallError:
1400:        except InstallError:
1527:        except InstallError:
1656:        except CmdError as e:
1811:        except Exception as e:
1814:            except Exception:
1885:        except HTTPError as e:
1910:        except tarfile.ReadError as e:
```

Comment (dcermak) [reply install.py]: python-language-server reported this issue and I was too lazy to check where it came from. This actually comes from pylint, the issue is though: pylint applies stricter rules to variable names, so `e` is a valid name according to PEP8 but pylint doesn't like it (too short). See: https://stackoverflow.com/questions/21833872/why-does-pylint-object-to-single-character-variable-names

I can change the name back to `e` as it is actually pretty clear what it is.

Comment (junaruga) [reply install.py]: Sure. Thanks for the URL. That's good enough.
What is the "python-language-server"?

I am fine to change `e` to `exec` if you can replace all the parts of `except ... as e` in target files (`install.py` and `tests/test_integration.py`).
But if it is hard to do it this time, I like that you remove your the last commit about the `e` for this PR.
Of course, you can send another PR. later.

Thanks.

Comment (dcermak) [reply install.py]: > Sure. Thanks for the URL. That's good enough.
> What is the "python-language-server"?

This: https://github.com/palantir/python-language-server. It's an implementation of the language server protocol (LSP) for python. LSP is a JSON based, editor agnostic protocol to provide code completion, jump to references, code lenses, etc. across various editors (emacs, vim, vs code, eclipse, etc). It's pretty neat: https://microsoft.github.io/language-server-protocol/.

> 
> I am fine to change `e` to `exc` if you can replace all the parts of `except ... as e` in target files (`install.py` and `tests/test_integration.py`).
> But if it is hard to do it this time, I like that you remove your the last commit about the `e` for this PR.
> Of course, you can send another PR later.

I'll update this PR.

> 
> Thanks.

Comment (dcermak) [reply install.py]: Done

**Pull request merged**